src/components/calculator/calculator.ts

```diff
@@ -101,7 +101,6 @@ export async function getCalculator(ctx: IContext, params: IParameters): Promise
     body: calculatorTemplate.render({
       routePartOf: ctx.routePartOf,
       linkTo: ctx.linkTo,
-      csrf: ctx.csrf,
```

Comment:
Can we not remove that line?

Though it's not being used, it's still parsed in the header over at: https://github.com/alphagov/paas-admin/blob/master/src/layouts/govuk.njk#L18
I would imagine on the long run it would be problematic to debug if the time comes.

Reply:
sure